pt-BR/mail/chrome/messenger/FilterEditor.dtd

```diff
@@ -57,31 +57,31 @@
  entities below were automatically added by l10nmerge
 -->
 
-<!ENTITY contextDesc.label "Apply filter when:">
-<!ENTITY contextManual.label "Manually Run">
-<!ENTITY contextBoth.label "Checking Mail or Manually Run">
-<!ENTITY contextIncoming.label "Checking Mail">
+<!ENTITY contextDesc.label "Aplicar filtro quando:">
+<!ENTITY contextManual.label "Executar manualmente">
+<!ENTITY contextBoth.label "Verificar emails ou executar manualmente">
```

Comment:
I disagree. Mail, in the context of an email application, is e-mail. There's no reason to change to mensagem, which is short for "mensagem de e-mail". E-mail is shorter, and this is important due to space limits.